third_party/kythe/generate_compilation_database.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Generates a compile_commands.json file at $(bazel info execution_root) for
+# your Clang tooling needs.
+
+BAZEL_ROOT="$(bazel info execution_root)"
+pushd "$BAZEL_ROOT" > /dev/null
```

Comment:
This and `popd` do not seem necessary? Just `cd "$BAZEL_ROOT"` should be enough?

Reply:
This script was just copied from Kythe. I'll change to `cd`.

Reply:
If it was just copied, feel free to leave it as it is (but leave a comment somewhere to say that it is copied). I thought you already modified it a bit, so you may as well modify it even more. TBH I think at this point you can just take it out of third_party and remove the license, it can just become one of our own scripts.